Source/WebKit/UIProcess/Launcher/glib/ProcessLauncherGLib.cpp

```diff
@@ -235,21 +235,37 @@ void ProcessLauncher::launchProcess()
         g_error("Unable to spawn a new child process: %s", error->message);
 
 #if OS(LINUX)
-    m_processID = IPC::readPIDFromPeer(pidSocketPair.server);
-    RELEASE_ASSERT(!close(pidSocketPair.server));
```

Comment:
Now that I look at this, why do we do this always on linux? Shouldn't we do this only when using bwrap sandbox?

Reply:
I suppose it's to keep only one code path in all cases, because this will always work regardless of using sandboxing or not, or which kind of sandbox. But's let's wait for @mcatanzaro to confirm.

Reply:
Adrian is correct. This is needed for bubblewrap sandbox and also flatpak sandbox, but not when running unsandboxed. It's harmless to do always, though. If we want to omit it when running unsandboxed, then the code would become a little more complicated; specifically, the auxiliary process would have a different number of command line arguments depending on whether we are sandboxed or not.